crates/nu-cli/tests/commands/sort_by.rs

```diff
@@ -22,6 +22,29 @@ fn by_column() {
     assert_eq!(actual.out, "description");
 }
 
+#[test]
+fn by_invalid_column() {
+    let actual = nu!(
+        cwd: "tests/fixtures/formats", pipeline(
+        r#"
+            open cargo_sample.toml --raw
+            | lines
+            | skip 1
+            | first 4
+            | split-column "="
```

Comment:
`split-column` is now `split column`.  You may need to pull in the latest master into your branch to see these changes.

Reply:
Thanks, didn't anticipate upstream changing this fast, lol! 💯